### ISSUE TITLE
No event binding after switching to alpha.19

### ISSUE THREAD

**Issue by SekibOmazic** [angular/angular#1359]
With alpha 19 event binding doesn't work any more. Same example works fine with alpha 18.

Plunk that demonstrates keyup behavior (type something into input field):

http://plnkr.co/edit/JpnVOTnLNPghydclKQcS?p=preview

**Comment by pkozlowski-opensource** [comment]
It looks like it is a NgElement that gets exposed on the local scope: http://plnkr.co/edit/Jc91BVTnKHTDBaWMFNhb?p=preview

**Comment by kosssi** [comment]
Add domElement:

```JS
@View({
  template: `
    <div>{{greeting}} world!</div><br/>
    <div>Enter something <input type="text" #ref (keyup) ></div>
    <span>you entered: {{ref.domElement.value}} !</span>
  `
})
```

**Comment by SekibOmazic** [comment]
@kosssi Yes, that's what @pkozlowski-opensource has found out in his plunk (see above). But to me taht is just a workaround.

**Comment by kosssi** [comment]
@SekibOmazic I juste would like add this information here ;)

**Comment by pkozlowski-opensource** [comment]
This was fixed in master by eac5c88893a5784384b6c7667945d7c21b94a48a
